rocketbelt/base/typography/_typography-cta.scss

```diff
@@ -4,9 +4,10 @@
   text-decoration: none;
   text-transform: uppercase;
   font-weight: font-weight(semibold);
+  line-height: 1.5;
   .icon {
-    width: .85em;
-    height: .85em;
+    width: .75rem;
```

Comment:
I lean hard toward keeping this in `em`s—that way if we use a huge CTA or a tiny one we don't have to update/override the size of the chevron.